Makefile

```
@@ -241,7 +242,10 @@ build-operator-image:
 	|| $(MAKE) docker-build docker-push
 
 build-operator-multiarch-image:
-	@ docker buildx imagetools inspect $(OPERATOR_IMAGE) | grep -q 'linux/arm64' 2>&1 >/dev/null \
+	@ hack/docker.sh -l -m $(OPERATOR_IMAGE)
+	@ hack/docker.sh -l -m $(OPERATOR_DOCKERHUB_IMAGE)
+	@ (docker buildx imagetools inspect $(OPERATOR_IMAGE) | grep -q 'linux/arm64' 2>&1 >/dev/null \
+	|| docker buildx imagetools inspect $(OPERATOR_DOCKERHUB_IMAGE) | grep -q 'linux/arm64' 2>&1 >/dev/null) \
```

I think it's `&&` to not call `docker-multiarch-build` only if both images exist. With `||`, we will consider it's OK as soon as an image exists.
```suggestion
	&& docker buildx imagetools inspect $(OPERATOR_DOCKERHUB_IMAGE) | grep -q 'linux/arm64' 2>&1 >/dev/null) \
```

---

Agreed, thanks.  Something is still up with the authentication in `hack/docker.sh` I'm working through.